README.md

```diff
@@ -122,6 +138,8 @@ Where `options` is an object and can contain the following:
     * Defaults to `datadoghq.com`.
     * See more details on setting your site at:
         https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site
+* `customServerURL`: Sends metrics to a custom server that supports Datadog metrics
+    format but is not datadog. This feature is incompatible with apiHost. (optional)
```

Comment:
This documentation needs the cover the environment variable.

I also think this needs some more explanation of how it and `apiHost` differ (not just that `apiHost` will be ignored if this is set). Why would I use one or the other? These two options are really similar, and I do not think it will always be obvious which someone should use.